src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -63,50 +63,50 @@ _* There are a few Nerdlets that basic users are allowed to use. See [Basic user
+If you're a basic user, you generally can't use any Nerdpacks. However, there are some Nerdlets that New Relic maintains that you can use. These are rare and there is currently no way for you to distinguish them in our web interface. And even if you can use a Nerdlet in a Nerdpack, you're never able to use custom visualizations. For this, you must be upgraded to a full platform user/core user.
 
 </Callout>
 
-### Full users
+### Full platform users/Core users
 
-If you're a full user in the original user model, you either:
+If you're a full platform user/core user in the original user model, you either:
 
 - Have the **Nerdpack Manager** add-on role
 - Don't have the **Nerdpack Manager** add-on role
 
 The **Nerdpack Manager** role is required to publish, subscribe to, and tag Nerdpacks. So if you don't have the **Nerdpack Manager** role, you can only create and serve Nerdpacks locally and use Nerdpacks that your accounts have already been subscribed to.
 
-To publish, tag, or subscribe to a Nerdpack, an admin must upgrade you to a full user or grant you the **Nerdpack Manager** add-on role.
+To publish, tag, or subscribe to a Nerdpack, an admin must upgrade you to a full platform user or grant you the **Nerdpack Manager** add-on role.
 
 ### Owner/Admin users
 
 If you're an owner or admin user, you can perform any of the Nerdpack capabilities. You can create, serve, publish, tag, subscribe to, and use any and all Nerdpack items, both Nerdlets and visualizations. This includes Nerdpacks built by New Relic or by one of your accounts.
 
 ## Nerdpack permissions: New Relic One user model
 
-Learn the differences between how basic users and full users can use and manage Nerdpacks in the [New Relic One user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/).
+Learn the differences between how basic users and full platform users can use and manage Nerdpacks in the [New Relic One user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/).
 
 ### Basic users
 
-If you're a basic user in the New Relic One user model, you're limited in your Nerdpack capabilities. You can only create and serve Nerdpacks locally. To publish, tag, subscribe to, and use your Nerdpacks, an admin must upgrade you to a full user.
+If you're a basic user in the New Relic One user model, you're limited in your Nerdpack capabilities. You can only create and serve Nerdpacks locally. To publish, tag, subscribe to, and use your Nerdpacks, an admin must upgrade you to a full platform user/core user.
```

Comment:
In the chart, the slash is fine because we want to save space. But in the actual text, we should use "or" because it reads better.

> ...an admin must upgrade you to either a full platform user or a core user.

And this applies to other places as well throughout this doc.